modules/oe_theme_content_project/config/overrides/core.entity_view_display.node.oe_project.default.yml

```diff
@@ -177,6 +180,14 @@ content:
       format_type: oe_project_date
       separator: '-'
     third_party_settings: {  }
+  oe_project_featured_media:
```

Comment:
Rendering test of this field has not been added to tests/Functional/ContentProjectRenderTest.php

Reply:
It can be implemented in PR #523 as method assertFeaturedMediaField() has been implemented there https://github.com/openeuropa/oe_theme/pull/523/files#diff-01cbad1c6adfd14f40e88b8361566814R513